components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -942,10 +956,10 @@ private class MonitoredItemWrapper {
             /// </summary>
             /// <param name="template"></param>
             /// <param name="logger"></param>
-            public MonitoredItemWrapper(MonitoredItemModel template, ILogger logger) {
+            public MonitoredItemWrapper(BaseMonitoredItemModel template, ILogger logger) {
```

Comment:
Can we add (or track) tests for this method? So we can be confident that the filters are right.

Reply:
I think we can cover this with unit tests for MonitoredItemModel. Will add in this PR